provider/k8s/controller_deployment.go

```diff
@@ -154,9 +155,9 @@ func (c *DeployController) SyncPDB(d *apps.Deployment, remove bool) error {
 		remove = true
 	}
 
-	minAvailable := &intstr.IntOrString{
+	pdb_default_min_available_percentage := &intstr.IntOrString{
 		Type:   intstr.String,
-		StrVal: "50%",
+		StrVal: os.Getenv("PDB_DEFAULT_MIN_AVAILABLE_PERCENTAGE")+"%",
```

Comment:
use this pattern for this: https://github.com/convox/convox/blob/master/provider/k8s/k8s.go#L157

don't take the value from env directly here.
also it's better to put validation on the value

Reply:
Added field in Provider struct for the variable and applied validation for a default value in case the env variable is nil.